pkgs/os-specific/linux/bluez/default.nix

```diff
@@ -56,7 +49,10 @@ stdenv.mkDerivation rec {
     "--with-udevdir=$(out)/lib/udev"
     ] ++ optional enableWiimote [ "--enable-wiimote" ]
       ++ optional enableMidi    [ "--enable-midi" ]
-      ++ optional enableSixaxis [ "--enable-sixaxis" ]);
+      ++ optional enableSixaxis [ "--enable-sixaxis" ]
+      ++ optional enableNfc     [ "--enable-nfc" ]
+      ++ optional enableMesh    [ "--enable-mesh" ]
+      ++ optional enableSap     [ "--enable-sap" ]);
```

Comment:
sap needs json_c as an additional dependency.

I've pushed a commit to https://github.com/Moredread/nixpkgs/tree/bluez which also adds another flag (`--enable-health`) and enables all optional flags for `bluezFull`.

I've sorted the flags, so it might fail to merge to the master branch though.

Feel free to use the commit as basis for your PR or squash it into yours. :)